api/routes/companies.js

```diff
@@ -0,0 +1,35 @@
+const router = require('express').Router()
+const Company = require('../models/company')
+
+// CREATE new company record
+
+router.post('/', (req, res, next) => {
+    const status = 201
+    Company.create(req.body).then(response => {
```

Comment:
It looks like this will work... but how is the Company going to be related to a specific unit/employees?